quic/s2n-quic-transport/src/connection/connection_impl.rs

```diff
@@ -1717,17 +1717,12 @@ impl<Config: endpoint::Config> connection::Trait for ConnectionImpl<Config> {
     }
 
     fn server_name(&self) -> Option<ServerName> {
-        // TODO move SNI to connection
```

Comment:
Is this comment important? Seems like we're not getting rid of the one at line 1726.

Reply:
The sni is now on space_manager in `space/mod` which is owned by the connection.